hreuniversalis/localisation/replace/VN_events_l_english.yml

```diff
@@ -2902,4 +2902,24 @@
  western_schism.8.b: "Adopt the Reformed faith"
  western_schism.8.c: "We must not apostatize!"
  western_schism.9.d: "The deep devotion of the Avignon Catholiques in Avignon has awakened a desire for an independent state ruled by Popes of the Church. A large part of the population has expressed support for the idea of such an establishment and has now turned to $MONARCH$ for approval."
- western_schism.9.b: "Avignon is rightfully theirs"
\ No newline at end of file
+ western_schism.9.b: "Avignon is rightfully theirs"
+#German Cultural Fragmentation
+ flavor_ger.9.pommeranian:0 "Let us integrate the §YPomeranians§!."
+ flavor_ger.9.austrian:0 "Let us integrate the §YAustrians§!."
+ flavor_ger.9.bavarian:0 "Let us integrate the §YBavarians§!."
+ flavor_ger.9.swiss:0 "Let us integrate the §YSwiss§!."
+ flavor_ger.9.swabian:0 "Let us integrate the §YSwabians§!."
+ flavor_ger.9.franconian:0 "Let us integrate the §YFranconians§!."
+ flavor_ger.9.saxon:0 "Let us integrate the §YSaxons§!."
+ flavor_ger.9.prussian:0 "Let us integrate the §YPrussians§!."
+ flavor_ger.9.hessian:0 "Let us integrate the §YHessians§!."
+ flavor_ger.9.hannoverian:0 "Let us integrate the §YWestphalians§!."
+ flavor_ger.9.lower_saxon:0 "Let us integrate the §YLower Saxons§!."
+ flavor_ger.9.central_margravian:0 "Let us integrate the §YMarkish§!."
+ flavor_ger.9.thuringian:0 "Let us integrate the §YThuringians§!."
+ flavor_ger.9.boden_alemannic:0 "Let us integrate the §YAlemannics§!."
+ flavor_ger.9.moselle_franconian:0 "Let us integrate the §YMoeselle Franconians§!."
+ flavor_ger.9.eastphalian:0 "Let us integrate the §YEastphalians§!."
+ flavor_ger.9.pfalzish:0 "Let us integrate the §YPfalzish§!."
+ flavor_ger.9.ripuarian:0 "Let us integrate the §YRhenish§!."
+ flavor_ger.9.baltic_german:0 "Let us integrate the §YBaltens§!."
```

Comment:
I'm not 100% sure on some of the plurals here -- "Alemannics"? And perhaps "Moeselle Franconians" should be "Moselles"? Unsure of the plural there too.

Let me know, happy to make changes.